xml/virt_docs.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter
+[
+  <!ENTITY % entities SYSTEM "generic-entities.ent">
+    %entities;
+]>
+
+<chapter xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0" xml:id="cha-virt-docs">
+ <title>Help and documentation</title>
```

Comment:
Maybe "Integrated Help and package documentation?" or something like that?
Plain "documentation" feels a bit a recursive self-reference to this very document.

Reply:
Reworded this a bit. Thanks for the suggestion!